_data/retail.yml

```diff
@@ -290,6 +297,14 @@ websites:
       facebook: northerntool
       email_address: intlenglishsales@northerntool.com
 
+    - name: Omura
+      url: https://shopomura.com
+      twitter: shop_omura
```

Comment:
img tag....

Reply:
I suppose `img` could be added until we confirm that it is no longer required, since we have our `generic.png` idea in place.

Reply:
he included an image though....

Reply:
My bad. In that sense, yes, it should be added. Appears I am not the best reviewer of YAML, but I will keep trying!

Reply:
whoops. fixed.